src/main/java/core/basesyntax/FileEditor.java

```diff
@@ -0,0 +1,279 @@
+package core.basesyntax;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class FileEditor {
+    public static void main(String[] args) {
+        InputStreamReader isr = new InputStreamReader(System.in);
+        BufferedReader reader = new BufferedReader(isr);
+
+        String fileData = "";
+        while (true) {
+            try {
+                String consoleLine = reader.readLine();
+                int commandSeparator = consoleLine.indexOf(" ");
+
+                String commandName;
+                if (commandSeparator != -1) {
+                    commandName = consoleLine.substring(0, commandSeparator);
+                } else {
+                    commandName = consoleLine;
+                }
+
+                ConsoleCommands command;
+
+                try {
+                    command = ConsoleCommands.valueOf(commandName.toUpperCase());
+
+                    if (commandSeparator != -1) {
+                        consoleLine = consoleLine.substring(commandSeparator);
+                    }
+                } catch (Exception e) {
+                    while (true) {
+                        int answer = getAnswerYesNo("Хотите сохранить данные в файл?");
+
+                        if (answer != 0) {
+                            if (answer == 1) {
+                                command = ConsoleCommands.CREATE;
+                                fileData = consoleLine;
+
+                                System.out.println("Введите путь к файлу и его имя для записи: ");
+                                consoleLine = reader.readLine();
+
+                            } else {
+                                command = ConsoleCommands.EXIT;
+                            }
+                            break;
+                        }
```

Comment:
do not build your logic using try catch. Make your own validation and separate method fo each case